test/modules/returns/lib/xml-to-json-mapping.js

```diff
@@ -0,0 +1,290 @@
+const { expect } = require('code');
+const { beforeEach, experiment, test } = exports.lab = require('lab').script();
+const { parseXmlFile } = require('../../../../src/modules/returns/lib/xml-helpers');
+const {
+  getReturnFrequency,
+  getNilReturn,
+  getMeterDetails,
+  wasMeterUsed,
+  getOverallReadingType,
+  getUnits,
+  getReadingDetails,
+  getReturnLines,
+  getEndDate,
+  getReadingType,
+  mapXml
+} = require('../../../../src/modules/returns/lib/xml-to-json-mapping');
+const path = require('path');
+const fs = require('fs');
+const util = require('util');
+
+const readFile = util.promisify(fs.readFile);
+
+experiment('XML to JSON Mapping', () => {
+  let returnXml;
+  let returnXmlNode;
+  let xmlFileToParse;
+
+  experiment('Completed Return', () => {
+    beforeEach(async () => {
+      xmlFileToParse = await readFile(path.join(__dirname, './xml-files-for-tests/single-monthly-return.xml'), 'utf-8');
+      returnXml = await parseXmlFile(xmlFileToParse);
+      returnXmlNode = returnXml.get('//tns:Return', { tns: 'http://www.environment-agency.gov.uk/XMLSchemas/GOR/SAPMultiReturn/06' });
+    });
+
+    test('getNilReturn function returns false', () => {
+      let xmlReturn = getNilReturn(returnXmlNode);
+
+      expect(xmlReturn).to.be.a.boolean().and.to.be.false();
```

Comment:
Checking if this is boolean is redundant becuase if false will fail if it is not.

`expect(xmlReturn).to.be.false();` does the job